proposals/0292-package-registry-service.md

```diff
@@ -557,8 +552,8 @@ consider the following global and local registry configuration files:
 ```jsonc
 // Global configuration (~/.swiftpm/config/registries.json)
 {
-  "registries": { 
-    "default": {
+  "registries": {
+    "[default]": {
```

Comment:
does square brackets have precedence in other such system? in Swift we usually use underbar for making "special" things

Reply:
I chose `[default]`, because it was visually distinctive. However, the only requirement is for it to not be a valid scope name, so `_default` could work just as well. At that point, it's a subjective choice.

As far as precedence, Swift uses square brackets in closures to annotate capture semantics (`[weak self]`), whereas underscore-prefixed variables are an unenforced convention. Another option to consider might be `_`. I don't feel too strongly either which way.

Reply:
I prefer we go with underbar `_` or `.`, tho the latter may be too subtle. @neonichu @abertelrud @yim-lee @weissi @lukasa opinions?

Reply:
+1 on `_`

Reply:
I think `_` is usually associated with private or "not to be used" things in the Apple developer community, so seems like we should avoid that.

Reply:
Agree with @neonichu , `_` means "don't use, the ACL on this decl is more permissive than it should ideally be" in most of Swift.

Happy with `[default]`, `<default>`, `(default)`, `.default`, ... Don't think I have a strong preference. We could also go with `""` to mean default but that's probably a bit opaque.

Reply:
Happy with any of @weissi's options there, I don't object to the `_` but I understand the desire not to use it.